packaging/ubuntu-16.04/rules

```diff
@@ -58,8 +58,16 @@ endif
 override_dh_fixperms:
 	dh_fixperms -Xusr/lib/snapd/snap-confine
 
+
+# The .real profile is a workaround for a bug in dpkg LP: #1673247 that causes
+# ubiquity to crash. It allows us to "move" the snap-confine profile from
+# snap-confine into snapd in a way that works with old dpkg that is in the live
+# CD image.
+#
+# Because both the usual and the .real profile describe the same binary the
+# .real profile takes priority (as it is loaded later).
 override_dh_installdeb:
```

Comment:
why is it done here, instead of override_dh_install?  the manpage says:  When using modern dh packaging techniques, dh_apparmor can be added to
       the override_dh_install section of the rules file.